config/config/sc-config.yaml

```
@@ -469,8 +469,12 @@ prometheus:
         limits:
           cpu: 10m
           memory: 100Mi
+  s3BucketPercentLimitAlarm: false
   s3BucketPercentLimit: 80
   s3BucketSizeQuotaGB: 1000
+  s3BucketNrOfObjectsAlarm: false
+  s3BucketNrOfObjects: 0
+  s3BucketsToExclude: []
```

If i may be super picky i do not like adding longer and longer names. Specially when working with yaml. So i would prefer these variables be a map with some good structure
 
```suggestion
  s3BucketAlerts:
  	limit:
  		enabled: false
  		percent: 80
  		sizeQuotaGB: 1000
  	nrOfObjects:
  		enabled: false
  		number: 80
		exclude: []
```